### ISSUE TITLE
Add subtitles to Mux videos on Sanity

### ISSUE THREAD

**Issue by iaradds** [sanity-io/sanity-plugin-mux-input#52]
Hi, we have Mux integrated with Sanity and currently only video and audio are supported but I was checking the documentation and you also offer the possibility of adding subtitles. Could you make this be available through the Sanity integration? Thank you

**Comment by simonhrogers** [comment]
Agree would be great to see this.

**Comment by michaelburtonray** [comment]
Agreed, Client requires subtitles in their videos as part of their accessibility guidelines.

**Comment by slegay** [comment]
+1

**Comment by thealice** [comment]
+1, this is a pretty important feature for a11y compliance.

**Comment by mfanuzzi** [comment]
+1! This is a major feature to be missing

**Comment by hdoro** [comment]
I'm happy to announce we're now working on the addition of auto-generated subtitles (follow #327 for updates on this).

As for custom subtitles uploaded via `.srt` or `.vtt` files, we're aware of its need and have prototyped ideas, but I can't promise a release date for a final solution.

Thank you all for the patience!

**Comment by ivyrze** [comment]
In case anyone still needs a workaround for this, I built [mux-captions](/ivryze/mux-captions) which you can use as a sidecar app for uploading captions. Hope this helps someone!
